osu.Game/Users/UserActivity.cs

```diff
@@ -236,15 +239,15 @@ public SpectatingUser() { }
         [MessagePackObject]
         public class SpectatingMultiplayerGame : InGame
         {
-            public SpectatingMultiplayerGame(IBeatmapInfo beatmapInfo, IRulesetInfo ruleset)
-                : base(beatmapInfo, ruleset)
+            public SpectatingMultiplayerGame(IBeatmapInfo beatmapInfo, IRulesetInfo rulesetInfo)
+                : base(beatmapInfo, rulesetInfo)
```

Comment:
Was the rename intentional? If so, it should be applied to the other constructors.

Reply:
seriously? I'll just revert it I guess